### PR TITLE
Adds generation of file tree

### PR DESCRIPTION
**Pull request by jamadaha**
This generates a file tree of the data set.

### CODE REVIEW

**Comment by kris701** [comment]
Why not just follow whatever the datasets structure is?
This will just require a lot of rewriting later, if we where to use other labels, would it not?

**Comment by jamadaha** [comment]
The reason behind this PR, is that it allows use of stuff like https://www.tensorflow.org/api_docs/python/tf/keras/utils/image_dataset_from_directory where it automatically imports from a file structure.

And I don't think it would require heavy rewrite later, as it builds from the generated CSV, rather than the existing file structure. So, as long as the CSV contains labels and paths, it would work.

**Review by kris701 — APPROVED** [review]
In that case, its fine
Worst case scenario, we can still use the CSV file as is
